recap/recap.py

```diff
@@ -637,44 +509,42 @@ def make_pdf_json(self, data, suffix=''):
         blocks = self.get_blocks_list(user, which_blocks)
         html = self.get_user_layout(blocks, user)
 
+        if self.css_file:
+            html = '<style>{}</style>{}'.format(self.css_file, html)
+
         if "<h3>" in html:
             html = re.sub("<h3>(.*?)<\/h3>","<h3>{}</h3>".format(data['document_heading']), html)
         else:
             html = u"<h3>{}</h3> \n {}".format(unicode(data['document_heading']), unicode(html))
 
         return {'html': html, 'user_name': user.username}
 
-
     @XBlock.json_handler
     def refresh_table(self, data, suffix=''):
         """ Complete HTML view of the XBlock, for refresh by client """
         course_id = self.location.course_key
         recap_blocks = self.get_recap_course_blocks(course_id)
         selected_recap_index = data["recap_id"]
 
-        recap_name_list = []
         block_ids = []
         for block in recap_blocks:
             block_ids.append(self.get_blocks_filtering_list(block.xblock_list))
-        
+
         query_list = []
         for block_id in block_ids[selected_recap_index]:
             query_list.append(Submission.objects.filter(
                 student_item__item_id=block_id
             ).values_list('student_item__student_id', flat=True))
 
-        student_ids_intersection = list(set.intersection(*map(set,query_list)))
-        
+        student_ids_intersection = list(set.intersection(*map(set, query_list)))
 
         downloadable_users = User.objects.filter(
             courseenrollment__course_id=course_id,
             courseenrollment__is_active=1,
             id__in=map(int, student_ids_intersection)
         ).values('username', 'email', 'id')
 
-        return  {"data": list(downloadable_users) }
-        
-
+        return {"data": list(downloadable_users)}
```

Comment:
Please, don't mix single quotes and double quotes. @andrey-canon

Reply:
changed